tests/loading_from_clipboard/loading_from_clipboard_test.cpp

```diff
@@ -0,0 +1,66 @@
+#include "gtest/gtest.h"
+#include "loading_from_clipboard_test.h"
+#include <QTextStream>
+#include "../../common/decklist.h"
+
+DeckList *fromClipboard(QString *clipboard);
+
+DeckList *fromClipboard(QString *clipboard) {
+    DeckList *deckList = new DeckList;
+    QTextStream *stream = new QTextStream(clipboard);
+    deckList->loadFromStream_Plain(*stream);
+    return deckList;
+}
+
+struct AssertDeckList {
+    QList<QString> expectedCards;
```

Comment:
Super-tiny consistency nit - you use `QList<QString>` here but `QStringList` later, which is an alias for the former.